### ISSUE TITLE
An error occurs when I attempt to execute "gsudo regedit"

### ISSUE THREAD

**Issue by septsea** [gerardog/gsudo#36]
(I set an alias for gsudo.)

![test1](https://user-images.githubusercontent.com/57337516/82958796-d8fe6080-9fe8-11ea-8956-578a16ec1b3d.gif)

```
Error: System.ComponentModel.Win32Exception (0x80004005): The operation completed successfully
   at gsudo.Helpers.ProcessFactory.CreateProcessAsUserWithFlags(String lpApplicationName, String args, CreateProcessFlags dwCreationFlags, PROCESS_INFORMATION& pInfo)
   at gsudo.ProcessRenderers.TokenSwitchRenderer..ctor(Connection connection, ElevationRequest elevationRequest)
   at gsudo.Commands.RunCommand.GetRenderer(Connection connection, ElevationRequest elevationRequest)
   at gsudo.Commands.RunCommand.<RunUsingElevatedService>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at gsudo.Commands.RunCommand.<Execute>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at gsudo.Program.<Start>d__1.MoveNext()
```

**Comment by Tiebe** [comment]
First of all, don't set an alias. It already support the sudo and gsudo commands. Or did you set the alias to something else than those 2?

Second, it works for me in PowerShell but doesn't seem to work in Command Prompt.

**Comment by septsea** [comment]
> @Tiebe 
> First of all, don't set an alias. It already support the sudo and gsudo commands. Or did you set the alias to something else than those 2?
> 
> Second, it works for me in PowerShell but doesn't seem to work in Command Prompt.

First, I installed gsudo by directly downloading the GitHub release, so I manually set an alias, It does not depend on the alias.

Second, `gsudo regedit` works in PowerShell but not in Command Prompt.

Actually there is no need to type such a command, because Windows 10 automatically pops up a window for privilege elevation if the user simply types `regedit`.

**Comment by gerardog** [comment]
I was able to reproduce it on CMD. With or without an alias.

This happens with the new default `TokenSwitch` mode, where the elevated process is created by the non-elevated gsudo, and the elevated instance replaces it's token with an elevated one. 

A few workarouds: Make gsudo elevate cmd which then runs regedit: 
`gsudo start regedit`
`gsudo cmd /c regedit`
  
Use attached mode instead of TokenSwitch:
`gsudo --attached regedit`

I am not sure yet what's the root cause here, or why this does not happen with PowerShell. Thanks @Tiebe for finding that last piece of info. It may lead to finding the root cause...

**Comment by gerardog** [comment]
The first step for `TokenSwitch` method is to create the process with `CreateProcess` API and some special flags. That is failing with `Regedit.Exe` because it has a manifest file embedded requesting elevation. (Thanks @septsea for finding out). 
And this is also happening with `gsudo eventvwr.exe` and probably with all [this list](https://www.researchgate.net/figure/List-of-auto-elevate-processes-in-Windows-10-after-executing-the-command-stringsexe-s_fig4_319454675)

**Comment by gerardog** [comment]
The fix was released in [v0.7.1](https://github.com/gerardog/gsudo/releases/tag/v0.7.1).
Thanks for reporting the problem.
